products/compute/terraform.yaml

```diff
@@ -268,6 +268,8 @@ overrides: !ruby/object:Provider::ResourceOverrides
       serviceLabel: !ruby/object:Provider::Terraform::PropertyOverride
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validateGCPName'
+      allPorts: !ruby/object:Provider::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
Computed: true (which is what default_from_api gives you) on booleans is almost never a good idea, since computed means "if this field is unset (which includes being set to the zero-value), don't diff it". This means that if the user changes it from true to false, then Terraform won't pick up the change, since false is the zero-value.

Reply:
Fixed.